### ISSUE TITLE
Auth to the web interface

### ISSUE THREAD

**Issue by spion-h4** [aroundus-inc/verdaccio-github-oauth#1]
Is it possible to authenticate to the verdaccio web interface using github oauth, or is it package-client-only?

**Comment by kimxogus** [comment]
Currently, it's not possible to authenticate in web interface.
Sorry for late response, github didn't send me email notification for this issue

**Comment by juanpicado** [comment]
ref https://github.com/verdaccio/verdaccio/pull/834 it's not the same?

**Comment by spion-h4** [comment]
Not 100% the same, that PR adds the capabilities needed for a plugin to provide a login link to the interface . The rest will probably need to be implemented in the plugin.

**Comment by spion-h4** [comment]
Specifically, the plugin will need to implement the same flow that ends up redirecting the CLI  to its own server [here](https://github.com/aroundus-inc/verdaccio-github-oauth/blob/c069be1cb991013e48928326a6b989d7aa97651a/index.js#L161) to instead set the proper auth JWT (get one via `auth.issueUIjwt`, then set it in local storage) and redirect to the website. Then it needs to inform verdaccio of the existance and address of that link via the new plugin api

**Comment by kimxogus** [comment]
Thanks for good information 👍 
As we are not using web interface, we don't have plan to implement github login in web interface for now.
Your PR is always welcomed!

**Comment by spion-h4** [comment]
Yep, I plan to submit a PR once verdaccio's support gets merged above.

**Comment by n4bb12** [comment]
Sounds like what https://github.com/n4bb12/verdaccio-github-oauth-ui does.
